edx/frontend-app-gradebook
# refactor - Bulk management tab unit tests

**TL;DR -** 
Cleans up BulkManagement component and adds docstrings and unit tests.

BulkManagement becomes BulkManagementTab (because that's what it is)
4 new sub-components: BulkManagementAlerts, FileUploadForm, HistoryTable, and ResultsSummary.

the `transfofrmHistoryEntry=>getRowsProcessed` pipeline is *only* used for this table, so I cleaned up its output to explicitly give what is needed for the results summary, instead of just passing the values necessary to derive said string, when the values themselves aren't actually needed.

Partial ticket completion
JIRA: [AU-3](https://openedx.atlassian.net/browse/AU-3)

**Developer Checklist**

**Testing Instructions**
The main place to check that is still working is the BulkManagement tab, and then just a smoke test of basic functionality.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:



FYI: @edx/masters-devs-gta
